### ISSUE TITLE
Casting should reset swing timer

### ISSUE THREAD

**Issue by Anonic** [watchyoursixx/WeaponSwingTimer-SixxFix#1]
Project lookin' a little abandoned, and no clue if it is even possible in Classic, but it seems like if you cast a spell the swing timer should be reset to needing a full swing time again.

**Comment by watchyoursixx** [comment]
I've been meaning to make improvements to it, but it takes me learning more of LUA code to understand how to change it. The fixes I did were pretty basic and I just wanted to get it functional.. I have a few things I'd like to add/fix going forward when I get time

**Comment by Anonic** [comment]
Yeah I hear that. Appreciate you fixing any of it to begin with, great lil swing bars.

**Comment by Zarant** [comment]
I've tried to tackle that issue before, the problem here is that certain spells reset the swing timer and some other spells don't, you can't really tell if a spell will reset the swing timer or not through the WoW API

**Comment by Anonic** [comment]
I noticed that as I got further on the Druid. Just one of those things I guess.
